### PR TITLE
Add lb http response rule

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: nsxt/lb_rule_utils.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+				},
+				"case_sensitive": getLbRuleCaseSensitiveSchema(),
+				"match_type":     getLbRuleMatchTypeSchema(),
+			},
+		},
+	}
+}
+
+func getLbRuleHTTPHeaderConditionSchema() *schema.Schema {
+	return &schema.Schema{
```
Comment (asarfaty) [inline nsxt/lb_rule_utils.go]: Max 1 entry?

File: nsxt/lb_rule_utils.go [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+				},
+				"case_sensitive": getLbRuleCaseSensitiveSchema(),
+				"match_type":     getLbRuleMatchTypeSchema(),
+			},
+		},
+	}
+}
+
+func getLbRuleHTTPRequestMethodConditionSchema() *schema.Schema {
+	return &schema.Schema{
```
Comment (asarfaty) [inline nsxt/lb_rule_utils.go]: Max 1 entry?

File: nsxt/resource_nsxt_lb_http_request_rewrite_rule.go [hunk 3]
```diff
@@ -639,8 +401,14 @@ func resourceNsxtLbHTTPRequestRewriteRuleRead(d *schema.ResourceData, m interfac
 	d.Set("display_name", lbRule.DisplayName)
 	setTagsInSchema(d, lbRule.Tags)
 	setLbRuleHTTPRequestConditionsInSchema(d, lbRule.MatchConditions)
-	setLbRuleRewriteActionsInSchema(d, lbRule.Actions)
 	d.Set("match_strategy", lbRule.MatchStrategy)
+	err = setLbRuleRequestRewriteActionsInSchema(d, lbRule.Actions)
+
+	if err != nil {
+		log.Printf("[DEBUG] Failed to set action in LoadBalancerRule %d: %v", id, err)
```
Comment (asarfaty) [inline nsxt/resource_nsxt_lb_http_request_rewrite_rule.go]: 'id' needs %v and not %d

File: nsxt/resource_nsxt_lb_http_response_rewrite_rule.go [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+	}
+	d.Set("revision", lbRule.Revision)
+	d.Set("description", lbRule.Description)
+	d.Set("display_name", lbRule.DisplayName)
+	setTagsInSchema(d, lbRule.Tags)
+	setLbRuleHTTPResponseConditionsInSchema(d, lbRule.MatchConditions)
+	d.Set("match_strategy", lbRule.MatchStrategy)
+	err = setLbRuleResponseRewriteActionsInSchema(d, lbRule.Actions)
+	if err != nil {
+		log.Printf("[DEBUG] Failed to set action in LoadBalancerRule %d: %v", id, err)
```
Comment (asarfaty) [inline nsxt/resource_nsxt_lb_http_response_rewrite_rule.go]: 'id' needs %v or %s and not %d

File: nsxt/lb_rule_utils.go (hunk 1 above)
Comment (annakhm) [reply nsxt/lb_rule_utils.go]: No, multiples are allowed

File: nsxt/lb_rule_utils.go (hunk 2 above)
Comment (annakhm) [reply nsxt/lb_rule_utils.go]: Multiples are allowed (for example, not PUT and not POST)

File: nsxt/resource_nsxt_lb_http_request_rewrite_rule.go (hunk 3 above)
Comment (annakhm) [reply nsxt/resource_nsxt_lb_http_request_rewrite_rule.go]: Thanks, done

**Pull request merged**